platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/config/VirtualMapConfig.java

```diff
@@ -60,6 +60,10 @@
  * 		The interval between flushing of copies. This value defines the value of N where every Nth copy is flushed. The
  * 		value must be positive and will typically be a fairly small number, such as 20. The first copy is not flushed,
  * 		but every Nth copy thereafter is.
+ * @param copyFlushThreshold
+ *      Virtual root copy flush threshold
```

Comment:
Might be helpful to document these two like the other values, with a bit more detail of how to set a good value for a particular configuration.

Reply:
Agree. I added JavaDoc for both in the latest commit.